typedoc.json

```diff
@@ -21,5 +21,7 @@
     "propertyMembersFormat": "table",
     "interfacePropertiesFormat": "table",
     "fileExtension": ".mdx",
-    "entryFileName": "index"
+    "entryFileName": "index",
+    "theme": "kit-docs-theme",
+    "useCodeBlocks": true
```

Comment:
This implies that links go away. There's a future where we could format these signature code blocks as _Twoslash_ code blocks, but we'd need to walk the types and figure out what imports are required to make that work.